packages/frint-compat/src/createApp.spec.js

```diff
@@ -317,11 +323,12 @@ describe('frint-compat › createApp', function () {
     expect(foo).to.equal('bar');
   });
 
-  it('throws error if setRegion is called without a root app', function () {
-    const widget = new WidgetApp();
+  // NOTE: setRegion will never be called in v1.x
+  // it('throws error if setRegion is called without a root app', function () {
+  //   const widget = new WidgetApp();
 
-    expect(() => widget.setRegion('sidebar')).to.throw(/No root app instance available/);
-  });
+  //   expect(() => widget.setRegion('sidebar')).to.throw(/No root app instance available/);
```

Comment:
I moved all the tests of frint v0.x to `frint-compat` and kept them unchanged to make sure we can depend on `frint-compat` reliably. 

If I really had to remove any individual test, I added a `NOTE` on top of the commented block explaining why it's not needed any more. 

Just to keep a reference.